pkg/pillar/types/locationconsts.go

```diff
@@ -64,4 +64,6 @@ const (
 	CertObj = "cert.obj"
 	//ITokenFile contains the integrity token sent in attestation response
 	ITokenFile = "/var/run/eve.integrity_token"
+	//EveVersionFile contains the running version of EVE
+	EveVersionFile = "/run/eve-release"
```

Comment:
Should we use this definition elsewhere in the code which access the file?

Reply:
Forgot to reply to your comment, yes, I have already updated the other places.